my-movie-list/src/services/localstorage.tsx

```diff
@@ -0,0 +1,7 @@
+export const loadMovies = () => {
+    let movies: Movie[];
+
+    movies = JSON.parse(localStorage.getItem("movies") || "");
+
+    return movies;
+};
```

Comment:
Acabou não usando isso, né? Vai uma dica aqui: sempre coloque o `JSON.parse()` dentro de um `try..catch`.
Se o parsing der erro e não for tratado, quebra todo o fluxo e, possivelmente, a aplicação.